controllers/v1/solutionsController.js

```diff
@@ -1235,5 +1236,61 @@ module.exports = class Solutions extends Abstract {
       }
     });
   }
+
+  /**
+    * @api {get} /assessment/api/v1/solutions/getObservationSolutionLink?observationsolutionId:observationsolutionId&appName:appName 
+    * @apiVersion 1.0.0
+    * @apiName Observation Share Link
+    * @apiGroup Solutions
+    * @apiHeader {String} X-authenticated-user-token Authenticity token
+    * @apiParam {String} observationsolutionId Observation Solution External ID.
+    * @apiParam {String} appName Name of App.
+    * @apiSampleRequest /assessment/api/v1/observations/getObservationSolutionLink?observationsolutionId=Mid-day_Meal_SMC_Observation-Aug2019&appName=samiksha
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    {
+      "message": "Observation Link fetched successfully",
```

Comment:
@priyanka-TL  Observation solution link fetched successfully should be the message.